docs/user/custom-resources/apigateway/apigateway-v1beta2-adr.md

```diff
@@ -0,0 +1,153 @@
+| **timeout**                     |  **NO**   | Specifies the timeout for HTTP requests in seconds for all Access Rules. The value can be overridden for each Access Rule. The maximum timeout is limited to 3900 seconds (65 minutes). </br> If no timeout is specified, the default timeout of 180 seconds applies.                                                                        |
+| **rules**                       |  **YES**  | Specifies the list of Access Rules.                                                                                                                                                                                                                                                                                                          |
+| **rules.service**               |  **NO**   | Services definitions at this level have higher precedence than the Service definition at the **spec.service** level.                                                                                                                                                                                                                         |
+| **rules.path**                  |  **YES**  | Specifies the path of the exposed Service.                                                                                                                                                                                                                                                                                                   |
+| **rules.methods**               |  **NO**   | Specifies the list of HTTP request methods available for **spec.rules.path**. The list of supported methods is defined in [RFC 9910: HTTP Semantics](https://www.rfc-editor.org/rfc/rfc9110.html) and [RFC 5789: PATCH Method for HTTP](https://www.rfc-editor.org/rfc/rfc5789.html).                                                        |
+| **rules.mutators**              |  **NO**   | Specifies the list of the Istio mutators. Currently, the `Headers` and `Cookie` mutators are supported.                                                                                                                                                                                                                                      |
+| **rules.accessStrategies**      |  **YES**  | Specifies the list of access strategies. Supported are `no_auth`, `extAuth` and `jwt` as [Istio](https://istio.io/latest/docs/tasks/security/authorization/authz-jwt/) access strategy.                                                                                                                                                      |
```

Comment:
Here no_auth is still part of the access strategies, but in the noAuth example it is a field on the rule level. 
We need to update it here as well. Remove it from access strategies and add it to the rule-level.